demo/gallery/vat-alice.js

```diff
@@ -203,6 +206,25 @@ function makeAliceMaker(E, log) {
             }`,
           );
         },
+        async doTapFaucetAndSell(host) {
+          log('++ alice.doTapFaucetAndSell starting');
+          const pixelPaymentP = E(gallery).tapFaucet();
+          const { pixelIssuer, dustIssuer } = await E(gallery).getIssuers();
+          const exclusivePixelPaymentP = await E(pixelIssuer).getExclusiveAll(
+            pixelPaymentP,
+          );
+          const amount = await E(exclusivePixelPaymentP).getBalance();
+          // sellToGallery creates a escrow smart contract with the
+          // terms of the amount parameter plus what the gallery is
+          // willing to offer for it
+          // sellToGallery returns an invite to the smart contract
+          const inviteP = E(gallery).sellToGallery(amount);
+          const seatP = E(host).redeem(inviteP);
+          E(seatP).offer(exclusivePixelPaymentP);
+          const dustPurseP = E(dustIssuer).makeEmptyPurse();
+          const pixelPurseP = E(pixelIssuer).makeEmptyPurse();
+          return collect(seatP, dustPurseP, pixelPurseP, 'alice escrow');
```

Comment:
If I'm reading this right, the pixel proceeds are stored in pixelPurseP, which was created here, and never returned anywhere. Doesn't this lose the pixels at this point?

Reply:
Sorry, this wasn't where I wanted to add this comment. See later.